crates/aptos-protos/proto/aptos/remote_executor/v1/network_msg.proto

```diff
@@ -0,0 +1,20 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.remote_executor.v1;
+
+message NetworkMessage {
+    string sender_addr = 1;
+    bytes message = 2;
+    string message_type = 3;
+}
+
+message Empty {
+
+}
+
+service RemoteExecution {
```

Comment:
The name should be generic.. something like `NetworkingService`

Reply:
I will use 'NetworkMessageService'